src/volunteers.html

```diff
@@ -188,13 +189,50 @@ <h2 class="section-heading">Newsletter</h2>
         </li>
       </ul>
     </nav>
+
     <footer>
-      <div class="content">
-        <p> Copyright &#xA9; Women Hack For Non-Profits 2016.</p>
-        <p><a class="text-link" href="contact.html">Contact Us</a> - <a class="text-link" href="faq.html">FAQ</a></p>
+      <div class="row no-margin"> 
+        <div class="col-xs-12 col-md-8 position central">
+          <div id="site-map" class="col-xs-12 col-sm-4">
+            <h5>Site map</h5>
+            <ul>
+              <li><a href="about.html">About</a></li>
+              <li><a href="volunteers.html">Volunteers</a></li>
+              <li><a href="non-profits.html">Non-Profits</a></li>
+              <li><a href="projects.html">Projects</a></li>
+              <li><a href="events.html">Events</a></li>
+            </ul>
+          </div>
+
+          <div id="know-us" class="col-xs-12 col-sm-4">
+            <h5>Get to know us</h5>
+            <ul>
+                <li><a href="https://twitter.com/WHFNP"><i class="fa fa-twitter" aria-hidden="true"></i> Twitter</a></li>
+                <li><a href="https://www.facebook.com/whfnp/"><i class="fa fa-facebook" aria-hidden="true"></i> Facebook</a></li>
+                <li><a href="https://medium.com/@WHFNP"><i class="fa fa-medium" aria-hidden="true"></i> Medium</a></li>
+                <li><a href="https://github.com/womenhackfornonprofits"><i class="fa fa-github" aria-hidden="true"></i> Github</a></li>
+            </ul>
+          </div>
+
+          <div id="get-involved" class="col-xs-12 col-sm-4">
+            <h5>Get Involved</h5>
+            <ul>
+              <li><a href="volunteer-contact.html">Volunteer with us</a></li>
+              <li><a href="non-profits-contact.html">Work with us</a></li>
+            </ul>
+          </div>
+
+          <div class="col-xs-12">
+            <div class="col-xs-12 col-md-7 position central text-center">
+              <p>Copyright &#xA9; Women Hack For Non-Profits 2016.</p>
+            </div>
+          </div>
+        </div>
       </div>
     </footer>
-      <script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
bootstrap is not depended on jquery, if we re not using any of the javascript plugins and only using css/sass it is not needed.